pypeit/core/wavecal/identify.py

```diff
@@ -0,0 +1,770 @@
+""" Module for finding patterns in arc line spectra
+"""
+import os
+import copy
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+from matplotlib.colors import LinearSegmentedColormap, Normalize
+from matplotlib.cm import ScalarMappable
+import matplotlib.transforms as mtransforms
+from matplotlib.widgets import Button, Slider
+
+matplotlib.use('Qt5Agg')
+
+from pypeit.par import pypeitpar
+from pypeit.core.wavecal import fitting, waveio, wvutils
+from pypeit import msgs
+
+
+class Identify(object):
+    """
+    Manually identify arc lines.
```

Comment:
Can you add a bit more?  Am trying to guess whether this is 
part of a stand-alone script (i.e. outside the main run) or if it
is part of the main run.  Maybe it's both.  :)